deploy/kustomization.yaml

```diff
@@ -1,7 +1,6 @@
 images:
 - name: controller
-  newName: quay.io/periklis/multicluster-observability-addon
-  newTag: 0.0.1
+  newName: ttl.sh/acmiblancasa
```

Comment:
@JoaoBraveCoding We should fix this reference to use our rhobs quay repo and neither mime nor someone else's.

Reply:
But this has to be pushing to that repository, right? Won't we hit rate limits or start to push over each other's images?

Reply:
Maybe we could set something up to use env vars for this.

Reply:
So nowadays we are already pushing images to `rhobs/multicluster-observability-addon` when things get merged to main. The goal of changing this is only to cover the use-cases where someone wants to try the addon. For the development cycle, it's fine if we come up with a new way of doing things that doesn't change this file, but I would make that its own PR